### ISSUE TITLE
Есть ли возможность создавать фильтры, типа дропа?

### ISSUE THREAD

**Issue by buchkovsky** [sleeping-owl/admin#92]
Возможно ли создавать фильтры дропдауны?

**Comment by sleeping-owl** [comment]
Если имеется в виду фильтры по какому-то столбцу, то в версии 3.0 можно, если использовать фильтры столбцов (http://sleeping-owl.github.io/v3/ru/Column_Filters.html#select).
![2015-05-18 14 08 46](https://cloud.githubusercontent.com/assets/9197310/7678725/0ea22ca8-fd5f-11e4-8b8a-bf9957ef49a3.png)

**Comment by buchkovsky** [comment]
да, именно это имелось в виду. Спасибо, попробуем расширить 2-ю версию под них, так как уже слишком много написано в проекте, чтобы переехать на 3-ю

**Comment by phpclub** [comment]
А как сделать так чтоб выводились сверху?

**Comment by phpclub** [comment]
И можно ли для фильтров применить Scope - чтоб фильтре выводилось только  deleted=0 к примеру?
